models/comment-model.js

```diff
@@ -16,3 +16,19 @@ exports.selectCommentsByReview = async (review_id) => {
   }
   return commentQuery.rows;
 };
+
+exports.insertComment = (review_id, updates) => {
+  const { username, body } = updates;
+  const d = new Date();
+  let time = d.getTime();
```

Comment:
You don't need to add a new date as SQL is doing this for you using the TIMESTAMP datatype